cmd/qos.go

```diff
@@ -74,6 +75,11 @@ func getServiceQoSInstances(
 		case config.ServiceIDPOKT:
 			// TODO_TECHDEBT: add pokt qos service here
 
+		case config.ServiceIDNoop:
+			if _, ok := gatewayServiceIDsIdx[serviceID]; ok {
+				gatewayQoSService[serviceID] = noop.NoOpQoS{}
+			}
+
 		default:
 			return nil, nil, fmt.Errorf("error building QoS instances: service ID %q not supported by PATH", serviceID)
```

Comment:
Why isn't the default QoS an instance of `noop.NoOpQoS{}` instead returning an error?
I would prefer a big warning and allow the user to proceed without any QoS module.

If I understand correctly the map of service IDs to QoS modules is hardcoded. If I stake a new service that is not in the list of the NoOpQoS, I would need to manually add my service to the NoOpQoS list and recompile PATH.